src/Private/Endpoints.ps1

```diff
@@ -385,4 +385,49 @@ function Get-PodeEndpointByName {
     }
 
     return $null
+}
+
+
+<#
+.SYNOPSIS
+    Displays information about the endpoints the Pode server is listening on.
+
+.DESCRIPTION
+    The `Show-PodeEndPointConsoleInfo` function checks the Pode server's `EndpointsInfo`
+    and displays details about each endpoint, including its URL and any specific flags
+    such as `DualMode`. It provides a summary of the total number of endpoints and the
+    number of general threads handling them.
+
+.EXAMPLE
+    Show-PodeEndPointConsoleInfo
+
+    This command will output details of all endpoints the Pode server is currently
+    listening on, including their URLs and any relevant flags.
+
+.NOTES
+    This function uses `Write-PodeHost` to display messages, with the `Yellow` foreground
+    color for clarity. It ensures each endpoint is displayed with its associated flags,
+    enhancing visibility of specific configurations like `DualMode`.
+#>
+function Show-PodeEndPointConsoleInfo {
+    if ($PodeContext.Server.EndpointsInfo.Length -gt 0) {
```

Comment:
I know you don't like indented code :) 
It's fault of my college days. The professor was obsessed with what in English is called Formal Method. Where any function has one entry point and one only exit. Otherwise the demonstration of the correctness was a nightmare. 
Anyway I haded it :D